driver/grpc.go

```diff
@@ -349,6 +349,46 @@ func (g *grpcStreamReader) close() error {
 	return nil
 }
 
+func (c *grpcCRUD) search(ctx context.Context, collection string, req *SearchRequest) (SearchResultIterator, error) {
+	// should we set transaction context here?
```

Comment:
Since this is non-transactional API, would removing context break anything on server?